README.Rmd

```diff
@@ -57,25 +81,6 @@ Set the following environment variables in your `.Renviron` file:
 
 These will be used by `{oysteR}` to authenticate with OSS Index, bumping up the amount of requests you can make.
 
-### Tests
-
-Our tests are in:
-
-- `tests/testthat`
-
-You can run tests in R like so:
-
-`devtools::test()`
-
```

Comment:
It seems like this section addresses how to run oysteR tests (as opposed to the other test section that you added, which discusses how to use oysteR and testthat to check for vulnerabilities within unit tests ... unless I am mistaken?) Just wondering why you removed this section since it seems to me like it would still provide value to contributors.

Reply:
In an R package, there is only a single place to put and run tests (in the test/ directory). Either a user knows about tests (in which case no comment is required) or they don't (these tests aren't a good place to start).

My thought was simplifying the README

Reply:
Ok, that's fair. As someone coming from other ecosystems/languages, there are often different test frameworks available, so it's often useful for a project README to specify how to run tests. However if R just has a single canonical way to run tests (TIL!) then you're right, it makes sense to simplify by removing redundant info.